js/web/lib/wasm/jsep/webgpu/ops/softmax.ts

```diff
@@ -73,8 +73,8 @@ const createSoftmaxProgramInfo = (input: TensorView, attributes: SoftmaxAttribut
       }
       ${shaderHelper.registerUniform('packedCols', 'i32').declareVariables(x, output)}
       ${shaderHelper.mainStart()}
-        let gindex = i32(global_id.x);
```

Comment:
I notice https://github.com/microsoft/onnxruntime/blob/main/js/web/lib/wasm/jsep/webgpu/ops/gemm.ts#L100-L101 is also using `global_id.x` instead of `global_idx`. Could you please also change that place?